### ISSUE TITLE
Octavia - MaaS fails on Octavia if there are multiple projects named 'admin'

### ISSUE THREAD

**Issue by wesmattson** [rcbops/rpc-maas#723]
When running maas playbooks, if Octavia is in the environment and there are 2 projects named 'admin' the playbooks fail with the following:

```
TASK [Ensure Octavia observer member role assignment] **********************************************************************************************************************
fatal: [controller3 -> localhost]: FAILED! => {
    "changed": false
}

MSG:

Multiple matches found for admin
```
I am trying to set an override to set octavia_project to the id of the correct admin project as a workaround. I'll report back if that works but otherwise I think we may need to update the octavia maas playbooks to account for this situation unless the answer is just "dont have two projects named admin" which I'd be okay with too.

**Comment by cemason** [comment]
I don't think this repo ever gets updated anymore so I am leaving a note for the next wayward soul who ventures upon this accursed issue - changing octavia_project to the ID rather than the name ("admin") does indeed seem to work.
